src/python/pants/backend/native/subsystems/native_build_step_settings.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.native.subsystems.utils.mirrored_target_option_mixin import \
+  MirroredTargetOptionMixin
+from pants.option.compiler_option_sets_mixin import CompilerOptionSetsMixin
+from pants.subsystem.subsystem import Subsystem
+from pants.util.memo import memoized_property
+from pants.util.meta import classproperty
+
+
+class NativeBuildStepSettings(CompilerOptionSetsMixin, MirroredTargetOptionMixin, Subsystem):
+
+  options_scope = 'native-build-step-settings'
```

Comment:
This is still pretty verbose. Putting the words "options" or "settings" in the names of options is probably superfluous.

In this case you're ending up with a subsystem instance scope named `native-build-step-settings.c-compile-settings`.